src/App.js

```diff
@@ -1,24 +1,216 @@
-import activities from './resources/schedule.json'
+import activities from "./resources/schedule.json";
 import dayjs from "dayjs";
+import isoWeek from "dayjs/plugin/isoWeek";
+import { useEffect } from "react";
+import { useState } from "react";
 
 function App() {
+  const currentDate = dayjs().format("DD MMMM");
+  const tomorrowsDate = dayjs().add(1, "day").format("DD MMMM");
+  dayjs.extend(isoWeek);
+
+  const [todaysActivityList, setTodaysActivityList] = useState([]);
+  const [tomorrowsActivityList, setTomorrowsActivityList] = useState([]);
+  const [restOfWeekActivityList, setRestOfWeekActivityList] = useState([]);
+  const [nextWeeksActivityList, setNextWeeksActivityList] = useState([]);
+  const [futureActivityList, setFutureActivityList] = useState([]);
+
+  function todaysActivities() {
+    const activityGroup = activities.filter(
+      (activity) => dayjs(activity.startTime).format("DD MMMM") === currentDate
+    );
+    setTodaysActivityList(activityGroup);
+  }
+
+  function tomorrowsActivities() {
+    const activityGroup = activities.filter(
+      (activity) =>
+        dayjs(activity.startTime).format("DD MMMM") === tomorrowsDate
+    );
+    setTomorrowsActivityList(activityGroup);
+  }
+
+  function restOfWeekActivities() {
```

Comment:
Do you think this whole function could be improved with the time left? I think it could get better.